pkg/minikube/bootstrapper/exec_runner.go

```diff
@@ -48,11 +48,26 @@ func (*ExecRunner) Run(cmd string) error {
 func (*ExecRunner) CombinedOutput(cmd string) (string, error) {
 	glog.Infoln("Run with output:", cmd)
 	c := exec.Command("/bin/bash", "-c", cmd)
-	out, err := c.CombinedOutput()
+	out, err := c.StdoutPipe()
+	defer out.Close()
```

Comment:
CombinedOutput also gets us Stderr, right? I think if you add that this should be fine.

Reply:
Updated with another approach, and redirected both Stdout & Stderr